### DISCUSSION TITLE
Faculty unable to re-verify even with verification of employment letter from HR - is there GitHub support staff to assist?

### DISCUSSION THREAD

**Discussion by ruthwest** [community/community#54565] [GitHub Education]
### Hi everyone,

Hello
I am a tenured, full professor at research university. Our campus does not include dates on our ID cards for faculty. I obtained a verification of employment letter from our campus HR as per the GitHub page requirements. There is no place to submit that in the reverificatoin interface and no way to re-verify without it. The form only allows submission of JPG files. I have attempted to contact GitHub support and they shared they are unable to assist. Is there anyone at GitHub available to assist to resolve this issue? 
Thanks for any assistance you can offer.

**Comment by Z-K-O** [comment]
I am having similar issues and they sent me here. This is supremely unhelpful. The community will not be able to fix these issues.

**Comment by ruthwest** [comment]
> ### Hi everyone,
> Hello I am a tenured, full professor at research university. Our campus does not include dates on our ID cards for faculty. I obtained a verification of employment letter from our campus HR as per the GitHub page requirements. There is no place to submit that in the reverificatoin interface and no way to re-verify without it. The form only allows submission of JPG files. I have attempted to contact GitHub support and they shared they are unable to assist. Is there anyone at GitHub available to assist to resolve this issue? Thanks for any assistance you can offer.
> 
> ### Detailed information in hopes this will get a reply
> _No response_

Hello again - i have not yet gotten any response from anyone on the GitHub education team to this question. Here is some additional information that may help to get this question noticed by the team.
I am a faculty member in CS and have been trying to re-verify for over three weeks. I have successfully reverified several times in the past, but with the new system, I am unable to do so. The deadline for re-verifying is soon approaching and I will soon need to remove all my student's work from GitHub. I have tried the following: I got a new ID card, I got a verification of employment letter from our HR department from the main campus HR department, I have tried to submit the link to our live campus faculty information center and nothing has worked. This has all taken many hours during end of semester and finals week. There is nothing further that I can do and I need assistance before this times out in a few days. Is there anyone that can assist? 
Thanks
